README-FR.md

```diff
@@ -0,0 +1,133 @@
+# if me
+
+  
+
+[if me](https://www.if-me.org/) est une communautée  sur les expériences sur la santé mentale qui encourage les gens a partager leur histoire personnel avec des personnes de confiance.
+les gens de confiance sont ceux avec qui on parle toues les jours , les membres de la famille, les amis, les professeurs, , et ceux qui travaille pour les expériences sur la santé.
+
+ 
+Traiter avec la santé mentale c'est ce qui fait de nous des humains. Mais pour beaucoup de nous, c'est un combat d'être ouvert à ce sujet. Pas tout le monde est un
+ conseiller ou un thérapeute
+les personnes avec lesquelles nous interagissons quotidiennement façonnent nos émotions et notre comportement.
+Les impliquer dans des traitements de santé mentale est la clé du rétablissement.
+  
+
+Le site  peut être trouvé à [if-me.org](https://www.if-me.org/).Les systèmes de conception en direct peuvent être trouvés à [design.if-me.org](http://design.if-me.org/).
+
+  
+
+On utilise le merveilleux[Contributor Covenant](http://contributor-covenant.org) pour notre code de conduite .  s'il vous plait
+
+[Lisez le](https://github.com/ifmeorg/ifme/blob/master/code_of_conduct.md)
+
+avant de rejoindre notre projet.
+
+  
+
+**Lisez à propos des objectifs de notre projet et comment y contribuer (not just as a developer) [ici](https://github.com/ifmeorg/ifme/blob/master/CONTRIBUTING.md).**
+
+  
+
+## Documentation
+
+  
+
+Regarder notre [Wiki](https://github.com/ifmeorg/ifme/wiki) pour avoir la documentation.
+
+  
+
+### Installation
+
+  
+
+les informations pour installer et configurer l'application [ici](https://github.com/ifmeorg/ifme/wiki/Installation). Les instances de test, de développement et de production sont couvertes.
+
+  
+
+### Contributeurs
+
+  
+
+Nous vous encourageons [a vous mettre](https://github.com/ifmeorg/ifme/wiki/Contributor-Blurb) sur la page des contributeurs.
+
+  
+
+## Donation
+
+  
+
+We also welcome financial contributions in full transparency on our
+On accepte aussi les contributions financières en toute transparence sur notre page
+[Open Collective](https://opencollective.com/ifme).
+
+Tout le modèle peut participer aux frais. Si la dépense a un sens pour le développement de  
+  
+la communauté, il sera "fusionné" dans le livre de notre Open Collective par le  
+  
+Les contributeurs principaux et les personnes qui ont effectué des dons seront remboursés.
```

Comment:
It should be `par les contributeurs principaux et les personnes qui ont effectué des dons seront remboursés.`